presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -156,6 +156,75 @@ Property Name                                      Description
 ``hive.hdfs.presto.keytab``                        HDFS client keytab location.
 ================================================== ============================================================ ==========
 
+Amazon S3 Configuration
+-----------------------
+
+The Hive connector also allows querying data stored in Amazon S3.
+
+To access tables stored in S3, you must specify the AWS credential properties 
+``hive.s3.aws-access-key`` and ``hive.s3.aws-secret-key``. Alternatively, you can use
+``hive.s3.use-instance-credentials`` which if set to true, enables retrieving temporary
+`instance profile <http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/java-dg-roles.html>`_
+AWS credentials.
+
+SQL Limitation for S3 tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The SQL support for S3 tables is the same as for HDFS tables. Presto does not support creating external
+tables in Hive (both HDFS and S3). If you want to create a table in Hive with data in S3, you have to do it from
```

Comment:
If you want to create a table -> if you want to create an "external" table. 

We should specify this limitation is for external tables only since S3 tables can also be managed hive tables.